response_operations_ui/static/css/main.css.map

```diff
@@ -1 +1 @@
tion: left center;\n    padding-left: 24px;\n    margin-left: -20px;\n}\n\n.btn--secondary.btn--border.btn--small {\n    margin: 0 0.5rem;\n    padding: 0.25rem 1rem;\n    width: auto;\n}\n\n.lock::before {\n    background: white url(\"/s/img/icons/icons--lock.svg\") no-repeat center center;\n}\n\n.page__content {\n    overflow-x: visible;\n}\n\n/* RU page Reporting unit page */\n\n.ru-survey {\n    margin: 1rem 0;\n    border: 0.5rem solid #eeecec;\n}\n\n.ru-survey h3 {\n    font-weight: 500;\n}\n\n.response__changed {\n    background-color: #edf4f0;\n}\n\n/* Collection exercise views */\n\n.ce-section h2 {\n    border-bottom: 2px solid #222;\n    padding-top: 9px;\n    margin-bottom: 0.3rem;\n    padding-bottom: 0.4rem;\n}\n\n.ce-event-date {\n    color: #666;\n    font-weight: 400;\n}\n\n.ce-event-date em {\n    color: #222;\n    font-weight: 500;\n}\n\ntable.ce-cis .tbl-ce-label {\n    vertical-align: middle;\n}\n\n.ci-info-panel {\n    margin: 2rem 0;\n}\n\n.ce-events__list {\n    width: 100%;\n    border-collapse: collapse;\n    margin: 1.3rem 0 0.3rem auto;\n}\n\n.ce-events__list th,\n.ce-events__list td {\n    text-align: left;\n    vertical-align: top;\n    padding: 0.5rem 0;\n    border-bottom: 1px solid #ccc;\n}\n\n.ce-events__list th {\n    font-weight: 700;\n    border-bottom: 1px solid #222;\n}\n\n.action-area {\n    margin-top: 1rem;\n    margin-bottom: 3rem;\n}\n\n#upload legend {\n    display: none;\n}\n\n#filedrag {\n    display: none;\n    font-weight: bold;\n    text-align: center;\n    padding: 1em 0;\n    margin: 1em 0;\n    color: #555;\n    border: 2px dashed #555;\n    border-radius: 7px;\n    cursor: default;\n}\n\n#filedrag.hover {\n    color: #0f8243;\n    border-color: #0f8243;\n    border-style: solid;\n    box-shadow: inset 0 3px 4px #888;\n}\n\n.ci-list {\n    border-collapse: collapse;\n    width: 100%;\n}\n\n.ci-list th,\n.ci-list td {\n    text-align: left;\n    padding: 0.5rem;\n}\n\n.ci-list th {\n    border-bottom: 1px solid #222;\n}\n\n/* Collection exercise - Reporting unit styles */\n\n.ceru-headline {\n    background: #f5f6ff;\n    padding: 1rem;\n    margin: 1rem 0;\n}\n\n.tempHack {\n    width: 2.5rem;\n    display: inline-block;\n}\n\n.error-message {\n    color: #d0021b;\n}\n\n/* Social */\n\n.cosy {\n    display: inline-block;\n}\n\n.case-search {\n    background-color: #e4e8eb;\n}\n\n.case-search-input2 {\n    width: 8rem;\n}\n\n.case-search-submit2 {\n    padding: 0.5rem 1rem;\n    margin-left: 0.5rem;\n}\n\n.case-search-results-info {\n    margin: 1rem 0 2rem 0;\n}\n\n.case-search-results-list {\n    list-style: none;\n    padding-left: 0;\n}\n\n.case-search-results-list__item {\n    margin-bottom: 1rem;\n}\n\n.results-list-ref {\n    margin-right: 1rem;\n}\n\n.results-list-postcode {\n    margin-right: 1rem;\n}\n\n.results-list-survey {\n    display: none;\n}\n\n.results-list-status {\n    display: none;\n}\n\n/* temporary styles for social */\n\n.case-address__prem {\n    display: block;\n}\n"]}
```

Comment:
There are still some references to the data panel within this file. though not sure how these are generated.

Reply:
interesting, this is the autogenerated file